fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,44 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+# If --dcmethod=NONE                                   # WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. 
+                                                       # This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of the 
+                                                       # distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with the cortical ribbon. 
+                                                       # In HCP data 30% of the cortical surface will be misaligned by at least half cortical thickness and 10% of the cortical 
+                                                       # surface will be completely misaligned by a full cortical thickness. At a future time, we may be able to add support for 
+                                                       # fieldmap-less distortion correction; however, no extant software package or pipelines have successfully demonstrated 
+                                                       # clear improvement over no distortion correction, in the direction towards gold standard fieldmap-based correction.
+
+
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+ProcessingMode=`opts_GetOpt1 "--processing-mode" $@`
+ProcessingMode=`opts_DefaultOpt $ProcessingMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+ComplianceWarn=""
+
+if [ "${DistortionCorrection}" = 'NONE' ]; then
+  ComplianceMsg+=" --dcmethod=NONE"
+  Compliance="LegacyStyleData"
+  log_Warn "WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of the distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with the cortical ribbon. In HCP data 30% of the cortical surface will be misaligned by at least half cortical thickness and 10% of the cortical surface will be completely misaligned by a full cortical thickness. At a future time, we may be able to add support for fieldmap-less distortion correction; however, no extant software package or pipelines have successfully demonstrated clear improvement over no distortion correction, in the direction towards gold standard fieldmap-based correction."
```

Comment:
For these long warning messages, I wonder if we should format as a bunch of shorter individual `log_Warn` lines, rather one single long line?  The ensuing result would then look something like:
```
<DateTime>: <ToolName>: WARNING: Line1 
<DateTime>: <ToolName>: WARNING: Line2
<DateTime>: <ToolName>: WARNING: Line3
```
as opposed to

`<DateTime>: <ToolName>: WARNING: One really long single line with too much wraparound to resolve by simply widening your viewing window...`

 I favor the former, as it should be both easier to read, and also highlights the "warning" nature by the fact that "WARNING" gets repeated on multiple lines.

Thoughts @coalsont ?

Reply:
My preference would be to have:

```
<DateTime>: <ToolName>: WARNING: Line1 
<DateTime>: <ToolName>:          Line2
<DateTime>: <ToolName>:          Line3
```

I am not sure, how to do that programatically, but I can split it up manually. How long should each warning line w/o the `<DateTime>: <ToolName>:` be?

Reply:
Let's wait for @coalsont's opinion before you do anything.  I'm not dogmatic about the line length (e.g., < 80 characters!), but I don't like it when the lines get so long that you can't prevent wraparound within a "reasonable" window width (say around 150-160 characters??)

Reply:
Also, FWIW, we already have other scripts that take the approach of just breaking long messages into multiple `log_Warn` lines.  Also, an advantage of that approach is that if we ever decided for example that `log_Warn` should report to stderr as well, you would still get the complete message.

Reply:
The date/tool part of the log messages is already quite long, so I don't really favor repeating it.  However, it is not clear that there is an easy solution that I would favor.  Maybe if I added the word-aware text wrapping with indenting from newopts.shlib to the logging shlib, this could be handled cleanly, but that isn't in scope for these PRs.

In other words: for now, whatever.

On a tangential note, I found it was easy to integrate testing for terminal width into the newopts.shlib word wrapping (though for redirection to files it may report a width of 80).

Reply:
So, since there is no easy solution, and since we already use multiple consecutive `log_Warn` lines in other places, that is the approach that I suggest we adopt here as well.

Reply:
Changed the warning.